Sampled/UI/VIews/LibraryInfoView.swift

```diff
@@ -272,36 +281,40 @@ struct LibraryInfoView: View {
           }
         }
 
-        LibraryInfoTagView {
-          LibraryInfoTagNameView {
-            Text("LibraryInfo.Tag.DiscNumber.Name")
-          }
-
-          LibraryInfoTagValueView {
-            LibraryInfoPositionTagView(number: libraryInfoTrack.discNumber, total: libraryInfoTrack.discTotal)
-          }
-        }
-
         LibraryInfoTagView {
           LibraryInfoTagNameView {
             Text("LibraryInfo.Tag.Duration.Name")
           }
 
           LibraryInfoTagValueView {
-            // It may be required to move away from this empty-value-mixed structure to display a mixed duration. We can
-            // compose it (e.g., having another property for mixed data), but I'm not sure if that's the best way to
-            // represent this. We could represent it like how Toggle's isMixed property is just an add-on to the general
-            // data.
             LibraryInfoTagValueContentView(property: libraryInfoTrack.duration) { duration in
               LibraryTrackDurationView(duration: duration)
             } mixed: {
-              LibraryInfoTagValueMixedNumberView()
+              if let total = libraryInfoTrack.totalDuration, let average = libraryInfoTrack.averageDuration {
+                VStack(alignment: .leading, spacing: 2) {
+                  HStack(spacing: 4) {
+                    Text("Total:")
+                      .foregroundStyle(.secondary)
+                    LibraryTrackDurationView(duration: total)
+                  }
+                  HStack(spacing: 4) {
+                    Text("Avg:")
+                      .foregroundStyle(.secondary)
+                    LibraryTrackDurationView(duration: average)
+                  }
+                }
+                .font(.caption2)
+              } else {
+                LibraryInfoTagValueMixedNumberView()
+              }
             }
           }
         }
       }
       .safeAreaPadding(.horizontal, 12)
+      .safeAreaPadding(.vertical, 8)
+      }
     }
-    .containerBackground(.ultraThickMaterial, for: .window)
+    .background(.ultraThickMaterial)
```

Comment:
I believe I used containerBackground to make sure that the material design would extend to the toolbar. Can you show what the UI looks like?